lib/readAlert.js

```diff
@@ -0,0 +1,46 @@
+require('enyo');
+require('enyo-webos');
+
+/**
+* Accessibility function for reading alert message. 
+*
+* @module enyo/readAlert
+* @public
+*/
+
+var
+	options = require('./options');
+
+module.exports = readAlert;
+
+function readAlert (s) {
+	if (!options.accessibility) return;
+
+	var isWebos = window.webOS && window.webOS.voicereadout && window.webOS.platform &&
+					window.webOS.platform.watch ||
+					window.webOS.platform.tv ||
+					null;
+	if (isWebos) {
+		window.webOS.voicereadout.readAlert(s);
+	} else {
+		removeAlertChild();
+		appendAlertChild(s);
+		setInterval(removeAlertChild, 300);
```

Comment:
Why setInerval call is needed? You should call clearInterval for clearing timer, or timer will accumulate whenever you call readAlert